platform/platform.go

```diff
@@ -105,6 +105,15 @@ func (conn *RestConnection) Refresh() error {
 		return err
 	}
 
+	if resp.StatusCode != 200 {
+		defer resp.Body.Close()
```

Comment:
But would the second `defer` even get called? It looks like the function would return early... :shrug:

Reply:
Yup, copy/paste error from functions below that typically return the io.ReadCloser stream. Fixing.

Reply:
So, defers get popped onto a stack and unwound when the function exits. So you can call `defer` anywhere in your function, and if the function exits anywhere after that `defer`, it will get called. Same goes for `recover` (more or less). So if you exit anywhere after that defer, you're good.